client/state/support-articles-alternates/selectors.js

```diff
@@ -14,7 +14,7 @@ export const isRequestingSupportArticleAlternates = ( state, postKeySegments ) =
 	const postKey = keyToString( postKeySegments );
 
 	return (
-		has( state.supportArticlesAlternates.requests, [ postKey ] ) &&
+		Object.hasOwn( state.supportArticlesAlternates.requests, postKey ) &&
 		isEmpty( get( state.supportArticlesAlternates.requests, [ postKey ] ) )
```

Comment:
Seems like this could be simplified - why do we really need the `hasOwn` check? Something like this could do:

```suggestion
		isEmpty( state.supportArticlesAlternates.requests[ postKey ] )
```

because `isEmpty()` will still work with `undefined` values.

Reply:
Hmm, I committed this and then immediately realized it's not a good idea: the selector is trying to distinguish between the state key being not present (before request) and being `{}` (after request and before completing)

Reply:
👍  I see. That's not ideal, but let's leave it as it was for the purpose of the PR.

Reply:
Reverted back :+1: